### PR TITLE
clear disk space after compaction

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md [hunk 1]
```diff
@@ -14,7 +14,15 @@ DROP SPACE [IF EXISTS] <graph_space_name>;
...
-`DROP SPACE`语句不会立刻删除硬盘上对应图空间的目录和文件，请使用`USE`语句指定其他任意图空间，然后执行`SUBMIT JOB COMPACT`。
+`DROP SPACE`语句不会立刻删除硬盘上对应图空间的目录和文件。
+
+如需删除无效空间的目录和文件：
+
+1. 在 Storage服务的配置文件中设置`auto_remove_invalid_space=true`。关于 Storage 服务的配置文件的路径，参见 [Storage 服务配置](../../5.configurations-and-logs/1.configurations/4.storage-config.md)。
+   
+2. 重启 Storage 服务。具体操作，参见[启动服务](../../4.deployment-and-installation/manage-service.md)。
+   
+3. 使用`USE`语句指定其他任意图空间，执行`SUBMIT JOB COMPACT`。
```
Comment (SuperYoko) [inline docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md:25]: Really? Please test without this step.

**Comment by SuperYoko** [comment]
# Test Report Based on v2.6.1

## Test Result for droped space directories/files.

### Use default config.

1. Create space use basketball script
2. Drop space basketball , and data is still there.
3. Restart nebula, and data is still there.
4. Create another space basketball2 and use and compact, space's data is still there. 
5. Drop basketball2, Create another space basketball3 and use and compact. both space's data is still there. 
6. add auto_remove_invalid_space and restart, all droped space's dir is deleted.

### Enalbe auto_remove_invalid_space [re-deployed.]

1. Create space use basketball script
2. Drop space basketball , and data is removed later.( no restart. )
3. Create 2 space and drop one , data is removed later.(no restart)

File: docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ DROP SPACE [IF EXISTS] <graph_space_name>;
 
 `IF EXISTS`关键字可以检测待删除的图空间是否存在，只有存在时，才会删除图空间。
 
-`DROP SPACE`语句不会立刻删除硬盘上对应图空间的目录和文件，请使用`USE`语句指定其他任意图空间，然后执行`SUBMIT JOB COMPACT`。
+`DROP SPACE`语句不会立刻删除硬盘上对应图空间的目录和文件。如需删除无效空间的目录和文件，在 Storage 服务的配置文件中设置`auto_remove_invalid_space=true`。关于 Storage 服务的配置文件的路径，参见 [Storage 服务配置](../../5.configurations-and-logs/1.configurations/4.storage-config.md)。
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md:17]:
```suggestion
`DROP SPACE`语句不会删除硬盘上对应图空间的目录和文件。如需删除无效空间的目录和文件，在 Storage 服务的配置文件中设置`auto_remove_invalid_space=true`。关于 Storage 服务的配置文件的路径，参见 [Storage 服务配置](../../5.configurations-and-logs/1.configurations/4.storage-config.md)。
```

**Review by SuperYoko — APPROVED** [review]
GJ.

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**